Doc/whatsnew/3.12.rst

```diff
@@ -79,6 +79,7 @@ New Features
 Other Language Changes
 ======================
 
+* The :class:`tempfile.NamedTemporaryFile` has a new optional parameter *delete_on_close* (Contributed by Evgeny Zorin in :gh:`22431`.)
```

Comment:
```suggestion
* The :class:`tempfile.NamedTemporaryFile` has a new optional parameter *delete_on_close* (Contributed by Evgeny Zorin in :bpo:`14243`.)
```

We typically refer to the issue number, not the pull request. That should also get around the (incorrect) error here.